src/core/hooks/createDataAttribute/createDataAttribute.test.tsx

```
@@ -0,0 +1,54 @@
+  );
+
+  // Render the hook that merges the two attribute objects into a single object
+  const { result: composedAttributes } = renderHook(() =>
+    useComposeAttributes(dataAttributes.current(), accentAttributes.current())
+  );
+
+  // Assert that the composed attributes object contains the expected `data-*` attributes
+  expect(composedAttributes.current()).toEqual({
+    "data-button-variant": "primary",
+    "data-button-size": "large",
+    "data-accent-color": "red",
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)